recipes/zulu-openjdk/all/test_package/conanfile.py

```diff
@@ -1,16 +1,27 @@
-from conans import ConanFile
+from conans import ConanFile, tools
 from six import StringIO
 
 
 class TestPackage(ConanFile):
+    settings = "os", "arch"
 
     def build(self):
         pass # nothing to build, but tests should not warn
 
     def test(self):
+        if self.settings.os == "Macos" and self.settings.arch == "armv8":
+            return
+        # if tools.cross_building(self):
```

Comment:
this should work , but does not, so I leave it here as a reminder

Reply:
I think it will work now, since you added `settings = "os", "arch"` now.
The latest c3i failure was 9e5f85a3c47feb47948ad8b81f8b66630a4e3897, but there you didn't add the settings object yet.

Reply:
thanks, fixed!